go.mod

```diff
@@ -13,12 +13,12 @@ require (
 	github.com/davecgh/go-spew v1.1.2-0.20180830191138-d8f796af33cc
 	github.com/fatih/color v1.15.0
 	github.com/go-git/go-git/v5 v5.11.0
-	github.com/hashicorp/go-getter v1.7.2
-	github.com/hashicorp/go-hclog v1.5.0
+	github.com/hashicorp/go-getter v1.7.4
+	github.com/hashicorp/go-hclog v1.6.2
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/hashicorp/hcl/v2 v2.17.1-0.20230725002108-58caf00be5aa
-	github.com/hashicorp/nomad v1.5.0-beta.1.0.20230804093607-388198abef93
-	github.com/hashicorp/nomad/api v0.0.0-20231219145541-859606a54ade
+	github.com/hashicorp/nomad v1.5.17
```

Comment:
not going higher?

Reply:
We only use it as a test dependency so I thought latest z of oldest supported version makes sense. I'm game to switch to whatever version we like.